Gemfile

```diff
@@ -14,4 +14,4 @@ gem "colorize", "~> 0.8.1"
 gem "cocoapods", "~> 1.9"
 gem "xcodeproj", "~> 1.18"
 gem "highline", "~> 2.0"
-gem "zip", "~> 2.0.2"
+gem "rubyzip", "~> 2.3.0"
```

Comment:
I had to replace it because I was getting an error: 
<img width="642" alt="Zrzut ekranu 2020-09-20 o 15 40 38" src="https://user-images.githubusercontent.com/4774319/93712675-a3f9b900-fb57-11ea-92e2-6e61b3269db5.png">

@facumenzella have you been experiencing it?